fh-android-sdk/pom.xml

```diff
@@ -26,6 +26,7 @@
   <properties>
     <loopj.version>1.4.8</loopj.version>
     <aerogear.android.push.version>2.2.1</aerogear.android.push.version>
+    <aerogear.android.store.version>3.0.0-SNAPSHOT</aerogear.android.store.version>
```

Comment:
> Any particular reason you can't use the latest stable version?

Yes, we changed the default database encryption from passphrase by keystore. You can see more information here => https://github.com/aerogear/aerogear-android-store/pull/34